x-pack/legacy/plugins/uptime/public/components/connected/alerts/alert_monitor_status.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectMonitorStatusAlert } from '../../../state/selectors';
+import { AlertMonitorStatusComponent } from '../../functional/alerts/alert_monitor_status';
+
+interface Props {
+  autocomplete: any;
+  enabled: boolean;
+  numTimes: number;
+  setAlertParams: (key: string, value: any) => void;
```

Comment:
no type for value?

Reply:
Yeah, per https://github.com/elastic/kibana/blob/master/x-pack/plugins/triggers_actions_ui/README.md#built-in-alert-types and https://github.com/elastic/kibana/blob/master/x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx#L127

The type of that handler isn't something we maintain, it comes from the alerting UI. I don't think the `triggers_actions_ui` exports a type for it but I can try to double check.

Reply:
Yes, we are planning to make it generic in the future, will makes sure about having a proper issue about that